webview-ui/src/components/prompts/PromptsView.tsx

```diff
@@ -798,7 +798,7 @@ const PromptsView = ({ onDone }: PromptsViewProps) => {
 												// Open or create an empty file
 												vscode.postMessage({
 													type: "openFile",
-													text: `./.roorules-${currentMode.slug}`,
+													text: `./.roo/rules-${currentMode.slug}/rules.md`,
```

Comment:
Seems like linking to a file with some default name was the easiest thing here, but let me know what you think!

Reply:
Indeed, it seems that having a default file name would be better!